projects/emissary-ingress/emissary/patches/0001-patching_helm_charts_packages.patch

```diff
@@ -6886,6 +6886,15 @@ index abe1dcaa2..cf5939e44 100644
    # Pod container image pull policy
    pullPolicy: IfNotPresent
  
+@@ -255,7 +257,7 @@ restartPolicy:
+ terminationGracePeriodSeconds:
+ 
+ waitForApiext:
+-  enabled: true
++  enabled: false
```

Comment:
I understand this is a workaround, but is this the intended behavior we want for the long term?

Reply:
Yeah, the idea is to keep it disabled by default to avoid this failure for users in air-gapped environment, and later we look into building the image and adding it in, if needed.

Reply:
But do we know what the side effects for removing this is?